pandas/core/generic.py

```diff
@@ -1834,6 +1835,42 @@ def to_hdf(self, path_or_buf, key, **kwargs):
             If applying compression use the fletcher32 checksum
         dropna : boolean, default False.
             If true, ALL nan rows will not be written to store.
+
+        See Also
+        --------
+        DataFrame.to_csv : write out to a csv file.
+        DataFrame.to_sql : write to a sql table.
+        DataFrame.to_feather : write out feather-format for DataFrames.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'A': [1, 2, 3], 'B': [4, 5, 6]}, 
+        ...                   index=['a', 'b', 'c'])
+        >>> df.to_hdf('data.h5', key='df', mode='w')
+
+        We can append another object to the same file:
+
+        >>> s = pd.Series([1, 2, 3, 4])
+        >>> s.to_hdf('data.h5', key='s')
+
+        Reading from HDF file:
+
+        >>> pd.read_hdf('data.h5', 'df')
+        A  B
+        a  1  4
+        b  2  5
+        c  3  6
+        >>> pd.read_hdf('data.h5', 's'))
+        0    1
+        1    2
+        2    3
+        3    4
+        dtype: int64
+
+        Notes
```

Comment:
I think Notes are before examples.

If you want to put a link, link to the section in the io.rst docs. You *can* add the HDF5 link in there if you want (in an appropriate location)

Reply:
Actually they already have link to the same manual there, so just removed this section.
Thanks!

Reply:
Also don't know why, but previously not all changes were pushed here, some remained locally, fixed it. Sorry for that.